README.md

```diff
@@ -1 +1,18 @@
 # hugegraph-hubble
+
+hugegraph-hubble A graph management and analysis platform that provides features: graph data load, schema management, graph relationship analysis and graphical display, and more.
```

Comment:
graph relationship analysis and graphical display.

deleting ", and more" is ok